firmware/application/apps/pocsag_app.cpp

```diff
@@ -179,7 +181,10 @@ void POCSAGAppView::on_packet(const POCSAGPacketMessage* message) {
             return;
         }
 
-        std::string console_info = "\n" + prefix;
+        // Indicate if the message has lots of decoding errors.
+        std::string color = pocsag_state.errors >= 3 ? "\x1B\x0D" : "";
```

Comment:
Perhaps someday we should put these escape strings in a header file (with a descriptive name indicating the color).